tests/integration/local/start_api/test_start_api.py

```diff
@@ -1521,22 +1521,32 @@ class TestServiceCorsSwaggerRequests(StartApiIntegBaseClass):
     def setUp(self):
         self.url = "http://127.0.0.1:{}".format(self.port)
 
-    @pytest.mark.flaky(reruns=3)
-    @pytest.mark.timeout(timeout=600, method="thread")
-    def test_cors_swagger_options(self):
-        """
-        This tests that the Cors are added to option requests in the swagger template
-        """
-        response = requests.options(self.url + "/echobase64eventbody", timeout=300)
-
+    def assertCors(self, response):
```

Comment:
nit: `snake_case`, the unittest stuff uses camel case, however the rest of our code use snake case.